spec/features/reviews/user_can_see_reviews_via_shelter_page_spec.rb

```diff
@@ -2,21 +2,22 @@
 
 RSpec.describe 'As a visitor' do
   it 'I can see shelter reviews via the shelter show page' do
-    review_1 = @shelter_1.reviews.create({ title: "This is Awesome",
-                  rating: "9",
-                  content: "This is some content about the shelter",
-                  image: 'https://i.imgur.com/wKls5bM.png'})
-    review_2 = @shelter_1.reviews.create({ title: "This is Great",
-                                           rating: "8",
-                                           content: "This is some content about the shelter",
-                                           image: 'https://i.imgur.com/wKls5bM.png'})
 
     visit "/shelters/#{@shelter_1.id}"
 
-    expect(page).to have_content(review_1.title)
-    expect(page).to have_content(review_1.rating)
-    expect(page).to have_content(review_1.content)
-    expect(page).to have_content(review_2.content)
-    save_and_open_page
+    expect(page).to have_content(@review_1.title)
+    expect(page).to have_content(@review_1.rating)
+    expect(page).to have_content(@review_1.content)
+    expect(page).to have_content(@review_2.content)
   end
+
```

Comment:
I say go ahead and add review without image to the before each block and have this implemented before we merge. Thoughts?